testing/environments/snapshot.yml

```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.11.0-SNAPSHOT
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.0.0-SNAPSHOT
```

Comment:
I'm afraid it won't work this way. First you need to upgrade the stack to support the Fleet Server. For reference: https://github.com/elastic/elastic-package/blob/master/internal/install/static_snapshot_yml.go

Reply:
It also seems to be a similar PR to @ycombinator 's one: https://github.com/elastic/package-storage/pull/1205

Reply:
I'll syncup with you sometime this week then we can see what's required to apply a bump automation in this project and the others